### PR TITLE
REGD-166-sidebar

### PR DESCRIPTION
**Pull request by mingjiecn**
What I did in this ticket:
1. Move navigation to header on every page.
2. Add cypress tests.

### CODE REVIEW

File: components/navigation.js [hunk 1]
```diff
@@ -201,479 +34,39 @@ NavigationLink.propTypes = {
...
-        <NavigationHrefItem
+      <div className="flex justify-end">
+        <NavigationLink
           id="contact"
-          navigationClick={navigationClick}
           href="mailto:regulomedb@mailman.stanford.edu"
-          target="_blank"
-          rel="noreferrer"
           aria-label="Email the RegulomeDB help desk"
+          isExternalLink={true}
```
Comment (forresttanaka) [inline components/navigation.js]: In JSX you can also just have `isExternalLink` by itself and it implies a value of true, if that’s more convenient.

File: components/navigation.js [hunk 2]
```diff
@@ -201,479 +34,39 @@ NavigationLink.propTypes = {
...
-        </NavigationHrefItem>
-        <NavigationHrefItem
+      <div className="flex justify-end">
+        <NavigationLink
           id="contact"
-          navigationClick={navigationClick}
           href="mailto:regulomedb@mailman.stanford.edu"
-          target="_blank"
-          rel="noreferrer"
           aria-label="Email the RegulomeDB help desk"
```
Comment (forresttanaka) [inline components/navigation.js]: I don’t think `aria-label` gets passed through to the link within `<NavigationLink>`. I think there’s a way to do that, but after all these years I’ve never looked into how (probably passing `{...props}` into the `<a>` would be my guess). I usually have a `label` property, then in the `<a>` use `aria-label={label}`.

**Review by forresttanaka — CHANGES_REQUESTED** [review]
Whether to change the `aria-label` depends on whether it works or not; I just doubt it works. Also, I think you should remove `breadcrumbs` from all your `getServerSideProps()`.

**Comment by forresttanaka** [comment]
Looks like you only run Cypress tests manually? It doesn’t seem like you have CircleCI tests configured, so I had thought you didn’t have Cypress tests.

File: components/navigation.js (hunk 2 above)
Comment (mingjiecn) [reply components/navigation.js]: got it.

**Comment by mingjiecn** [comment]
> Looks like you only run Cypress tests manually? It doesn’t seem like you have CircleCI tests configured, so I had thought you didn’t have Cypress tests.

Cypress tests are running in CircleCI: https://app.circleci.com/pipelines/github/RegulomeDB/regulome-ui/251/workflows/5422aaee-9c01-4527-aee8-da565583b90a/jobs/1242
Actually I forgot to push more tests. Will push them now.

**Comment by forresttanaka** [comment]
> Cypress tests are running in CircleCI

Oh, OK. The PR output didn’t look like it to me.

**Review by forresttanaka — APPROVED** [review]
Oh, you’re still using breadcrumbs. It looked like you weren’t at first.

**Pull request merged**